content/en/synthetics/browser_tests/actions.md

```diff
@@ -196,9 +200,19 @@ Create a variable out of a `span`, `div`, etc. content by extracting the text of
 
 ##### JavaScript
 
-Generate custom variables using your own JavaScript scripts. JavaScript steps support both synchronous and asynchronous code. By default the step is being performed on the active page, which means it can access all the objects defined at the active page level (libraries, builtins, global variables, ...).
+Generate custom variables using your own JavaScript scripts. JavaScript steps support both synchronous and asynchronous code.
+
+This JavaScript function comes with two parameters. You also need to make sure to include a `return` statement in your JavaScript snippet.
 
-If you need to load external libraries, you can wrap their loading in a promise like below:
+* `vars`: String containing your browser test [variables][5]. Use `vars.<YOUR_VARIABLE>` to refer to your browser test variable in your JavaScript snippet. Example: If your browser test contains a `USERNAME` variable, you can call it in your JavaScript snippet using `vars.USERNAME`.
+
+* `element`: Locator of the element on the page. This element can be selected and updated with the `Select` and `Update` target element buttons of the step. The selected element automatically leverages Datadog browser test multi locating algorithm.
+
+{{< img src="synthetics/browser_tests/js_variable.mp4" alt="Browser Test JavaScript Variable" video="true" width="100%">}}
+
+* The `return` statement needs to reflect the condition the assertion needs to meet for your browser test step to be successful. You can return any type, but the value is automatically casted as a string.
+
+By default the step is being performed on the active page, which means it can access all the objects defined at the active page level (libraries, builtins, global variables, ...). If you need to load external libraries, you can wrap their loading in a promise like below:
```

Comment:
```suggestion
Since JavaScript assertions run in the context of the active page, these steps can access all the objects defined in the active page (libraries, built-ins, global variables, etc.). To load external libraries, use a promise, for example:
```